src/auth/auth.controller.ts

```diff
@@ -37,4 +39,15 @@ export class AuthController {
   public async signUpWithEmailAndPassword(@Body() createUserDto: CreateUserDto) {
     return this.authService.signUpWithEmailAndPassword(createUserDto);
   }
+
+  @Get("google")
+  @UseGuards(AuthGuard("google"))
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  async googleAuth(@Req() _req: any) {}
```

Comment:
Remove the eslint comment and just add  a void return, and type `req` to use express `Request` object

`import { Request } from "express";`

```suggestion
  async googleAuth(@Req() _req: Request) {
      return;
  }
```

Reply:
Will do